vkconfig/layer_info.json

```diff
@@ -366,6 +366,35 @@
                 "type": "save_file",
                 "default": "stdout"
             }
+        },
+        "VK_LAYER_LUNARG_screenshot": {
+            "frames": {
+                "name": "Frames",
+                "description": "Comma separated list of frames to output as screen shots or a range of frames with a start, count, and optional interval separated by a dash. Setting the variable to \"all\" will output every frame. Example: \"5-8-2\" will output frame 5, continue until frame 13, dumping every other frame. Example: \"3,8-2\" will output frames 3, 8, and 9.",
+                "type": "string",
+                "default": "0-0"
+            },
+            "dir": {
+                "name": "Directory",
+                "description": "This can be set to specify the directory in which to create the screenshot files.",
+                "type": "string",
+                "default": ""
+            },
+            "format": {
+                "name": "Format",
+                "description": "This can be set to a color space for the output.",
+                "type": "enum",
+                "options": {
+                    "UNORM": "UNORM",
+                    "SNORM": "SNORM",
+                    "USCALED": "USCALED",
+                    "SSCALED": "SSCALED",
+                    "UINT": "UINT",
+                    "SINT": "SINT",
+                    "SRGB": "SRGB"
+                },
+                "default": "SRGB"
```

Comment:
In order to match the default (which appears to be to use the swapchain colorspace) where there's no env var set or the env var is the empty string, this may need to have an option "UNDEFINED" or "USE_SWAPCHAIN_COLORSPACE" or some such, which would be the default.  I am uncertain.  What do you think?

Reply:
I think the default should be "USE_SWAPCHAIN_COLORSPACE".
Also, I think there should be an added conditional to the function that gets the format that tells it not to print the error message if the user has the environment variable or settings set to "USE_SWAPCHAIN_COLORSPACE".

Reply:
Sounds good.